### ISSUE TITLE
genesis tool should allow users to skip the timestamp suffix behavior in output dir

### ISSUE THREAD

**Issue by luisnaranjo733** [ConsenSys/quorum-genesis-tool#42]
Currently when you run the genesis tool, it allows you to override the output path with user input, but it nests the output inside a subfolder which is named with an arbitrary timestamp (see [this ](https://github.com/ConsenSys/quorum-genesis-tool/blob/a06b56023578c2a2cda9d90a8dffd2d36e35568f/src/lib/networkGenerate.ts#L30)line of code)

This seems by design, but it would be great if there was another boolean flag that users could set to bypass this behavior such as ```--noOutputTimestamp```

By setting this flag, users could tell the genesis tool to skip that nested timestamp folder logic and instead output the files directly into the folder the user has already specified

The reason for this change is that the current default behavior makes it hard to consume the output automatically by a script, because that script would need to parse the STDOUT spew to find the subdir time stamp name

**Comment by luisnaranjo733** [comment]
Came up with some bash regex to parse this from the STDOUT spew
Not pretty but a decent workaround for now

```bash
# Generate genesis files, tessera keys, ethereum keys, and node keys
function generate_genesis() {
  echo "Generating genesis files..."
  npx quorum-genesis-tool \
  --outputPath ./output \
  --consensus qbft \
  --chainID 1337 \
  --blockperiod 5 \
  --requestTimeout 10 \
  --epochLength 30000 \
  --difficulty  1 \
  --gasLimit 0xFFFF \
  --coinbase 0x0000000000000000000000000000000000000000 \
  --maxCodeSize 64 \
  --txnSizeLimit 64 \
  --validators 0 \
  --members 4 \
  --bootnodes 0 \
  --accountPassword "" \
  --tesseraEnabled true \
  --tesseraPassword "" \
  --quickstartDevAccounts false
}


GENERATE_GENESIS_OUTPUT=$(generate_genesis)

if [[ $GENERATE_GENESIS_OUTPUT =~ (./output/[-[:digit:]]+)$ ]]; then 
  GENESIS_DIR=${BASH_REMATCH[1]}
  echo "Genesis files generated at $GENESIS_DIR"
else 
    echo "Failed to find the nested output dir from the genesis tool"
    exit 1
fi
```

**Comment by Ezzahhh** [comment]
Hi @luisnaranjo733, thank you for opening this issue. I have added your suggestion and released it as is in [0.2.9](https://github.com/ConsenSys/quorum-genesis-tool/releases/tag/0.2.9)

**Comment by luisnaranjo733** [comment]
Just reviewed the changes - this is exactly what I was hoping for

Thank you!
